app/src/main/java/org/schabi/newpipe/settings/VideoAudioSettingsFragment.java

```diff
@@ -58,10 +46,59 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
                             .show();
 
                 }
+            } else if (s.equals(getString(R.string.use_inexact_seek_key))) {
+                updateSeekOptions();
             }
         };
     }
 
+    /**
+     * Update fast-forward/-rewind seek duration options according to language and inexact seek setting.
+     * Exoplayer can't seek 5 seconds in audio when using inexact seek.
+     */
+    private void updateSeekOptions() {
+        //initializing R.array.seek_duration_description to display the translation of seconds
+        final Resources res = getResources();
+        final String[] durationsValues = res.getStringArray(R.array.seek_duration_value);
+        final List<String> displayedDurationValues = new LinkedList<>();
+        final List<String> displayedDescriptionValues = new LinkedList<>();
+        int currentDurationValue;
+        final boolean inexactSeek = getPreferenceManager().getSharedPreferences()
+            .getBoolean(res.getString(R.string.use_inexact_seek_key), false);
+
+        for (String durationsValue : durationsValues) {
+            currentDurationValue =
+                Integer.parseInt(durationsValue) / (int) DateUtils.SECOND_IN_MILLIS;
+            if (inexactSeek && currentDurationValue % 10 == 5) {
+                continue;
+            }
+
+            displayedDurationValues.add(durationsValue);
+            try {
+                displayedDescriptionValues.add(String.format(
+                    res.getQuantityString(R.plurals.seconds,
+                        currentDurationValue),
+                    currentDurationValue));
+            } catch (Resources.NotFoundException ignored) {
+                //if this happens, the translation is missing, and the english string will be displayed instead
+            }
+        }
+
+        final ListPreference durations = (ListPreference) findPreference(getString(R.string.seek_duration_key));
+        durations.setEntryValues(displayedDurationValues.toArray(new CharSequence[0]));
+        durations.setEntries(displayedDescriptionValues.toArray(new CharSequence[0]));
+        final int selectedDuration = Integer.parseInt(durations.getValue());
+        if (selectedDuration / (int) DateUtils.SECOND_IN_MILLIS % 10 == 5) {
```

Comment:
I just noticed an error in one of my PR, it needs to be
`if (inexactSeek && selectedDuration / (int) DateUtils.SECOND_IN_MILLIS % 10 == 5) {`
Otherwise the seek durations get rounded up, even if inexact seek is disabled

Reply:
Thanks. I'll fix it in a minute

Reply:
Fixed in https://github.com/TeamNewPipe/NewPipe/pull/3267/commits/55c577e76ec96fad3a52279eaea2201f8f05c2a7